polkadot/node/subsystem-bench/README.md

```diff
@@ -260,6 +260,35 @@ This file is best interpreted with `cg_annotate --auto=yes cachegrind.out.<pid>`
 
 For finer profiling of cache misses, better use `perf` on a bare-metal machine.
 
+### Profile memory usage using jemalloc
+
+Bellow you can find instructions how to setup and run profiling with jemalloc, this is complementary with using other memory profiling tools
+like: https://github.com/koute/bytehound?tab=readme-ov-file#basic-usage.
+
+#### Prerequisites
+Install tooling with:
+```
+sudo apt install libjemalloc-dev graphviz
```

Comment:
nit: I guess this only works for some Linux distributions, but I am a mac user :)

Reply:
Yes, linux only :D